gremlin-console/src/main/static/LICENSE

```diff
@@ -216,6 +216,10 @@ The Apache TinkerPop project bundles the following components under the BSD Lice
      minlog (com.esotericsoftware:minlog:1.3.0 - https://github.com/EsotericSoftware/minlog)
        - shaded in gremlin-shaded to org.apache.tinkerpop.shaded.minlog
        - for details, see licenses/minlog
+     Antlr4 (org.antlr:antlr4-runtime:4.9.1 - https://www.antlr.org) - for details, see licenses/antlr4
```

Comment:
the groovy jar LICENSE already references ANTLR (not the detail level we do, but i suppose that is their style). i don't think we need to do it again. it just complicates our LICENSE. simply including

Reply:
I'm certainly not an expert in this domain but my understanding was that for any dependency, including transitive ones, if any binary/source is packaged within our distribution then we need to meet any NOTICE/license obligations at the root level. If it is acceptable for those to be buried within the jar that contains the transitive dependency then ASM can definitely be removed here.

On the other hand the ANTLR jar is directly included in lib directory in both the console and server distributions. In this case I don't think the license buried within the groovy jar is good enough.

Reply:
oh...ok. then yeah for antlr then. 

for asm i think a less complicated NOTICE beats the buried one, but this is all an art. i tried to write "rules" as best i could in all my learnings from the ASF on how to do this, but i wouldn't say my "rules" are hard and fast. 

if you really want to dig into it, you could try to get an answer from the ASF. i'd guess the incubator would be the best place to ask this question. be prepared for 10 different answers though 🙂

Reply:
I have removed ASM licenses as they are properly covered by groovy. The BSD requirements are that redistributions must reproduce the license and it is definitely reproduced within the groovy jar. This argument could be extended to remove kryo and minilog from the console and server licenses as this PR is adding proper licensing for them within the gremlin-shaded jar.